### ISSUE TITLE
fotmob_get_match_details scrapes and unnests shots data

### ISSUE THREAD

**Issue by rowzreport** [JaseZiv/worldfootballR#166]
I am using the latest version: 0.5.12.1000. See below example and comments

```
#below function gives shot level data. 
#If I remember correctly (not 100% on this), it used to give main match info, and the last column was a shots dataframe. 
#it seems like the below function now just downloads the shots dataframe and unnests it
df1 = fotmob_get_match_details(3915296) ##sample matchid #1
df2 = fotmob_get_match_details(3932406) ##sample matchid #2
```

Output:
```
> head(df1[15:19])
# A tibble: 6 × 5
          id event_type   team_id player_id player_name  
       <dbl> <chr>          <int>     <int> <chr>        
1 2447448415 Miss            8655    820581 Ben Brereton 
2 2447450905 Goal            9798    789508 Tom McIntyre 
3 2447456503 AttemptSaved    9798   1123529 Tyrese Fornah
4 2447456983 AttemptSaved    9798    877567 Thomas Holmes
5 2447457197 Miss            9798    212441 Andy Yiadom  
6 2447457543 AttemptSaved    9798     23293 Shane Long   
```
>

**Comment by tonyelhabr** [comment]
Yes, this was a change in `0.5.8`. See the [NEWS.md](https://github.com/JaseZiv/worldfootballR/blob/main/NEWS.md#worldfootballr-058)

**Comment by tonyelhabr** [comment]
> Yes, this was a change in `0.5.8`. See the [NEWS.md](https://github.com/JaseZiv/worldfootballR/blob/main/NEWS.md#worldfootballr-058)

The reasoning: most people were using this function for the shot data anyways, so `unnest`ing for users is convenient.

**Comment by tonyelhabr** [comment]
But `fotmob_get_match_details()` didn't return match score, venue, referee, etc. before the unnesting change in `0.5.8`. (All the same columns + the unnested shot columns are returned with the existing `fotmob_get_match_details()` .) Are you asking for a new function? It wouldn't be hard I suppose.

**Comment by tonyelhabr** [comment]
> Ok fair - I was not 100% sure that match score, venue etc. was returned from `fotmob_get_match_details()`
> 
> In that regard, yes it would be great to have a new function then!

I can work on it. I'll leave this issue open until it's done.

**Comment by tonyelhabr** [comment]
Addressed with #173

**Comment by jterzioglu** [comment]
Thank you!
